pkg/standalone/standalone.go

```diff
@@ -688,12 +792,69 @@ func createRedisPubSub(redisHost string, componentsPath string) error {
 		return err
 	}
 
-	filePath := filepath.Join(componentsPath, pubSubYamlFileName)
-	err = ioutil.WriteFile(filePath, b, 0600)
+	filePath := path_filepath.Join(componentsPath, pubSubYamlFileName)
+	err = checkAndOverWrite(filePath, b)
+
+	return err
+}
+
+func createDefaultConfigurtion(configDir string) error {
+	defaultConfig := configuration{
+		APIVersion: "dapr.io/v1alpha1",
+		Kind:       "Configuration",
+	}
+	defaultConfig.Metadata.Name = "daprConfig"
+	defaultConfig.Spec.Tracing.SamplingRate = "1"
+
+	b, err := yaml.Marshal(&defaultConfig)
 	if err != nil {
 		return err
 	}
-        os.Chmod(filePath, 0644)
 
+	filePath := path_filepath.Join(configDir, defaultConfigFileName)
```

Comment:
Could use GetDefaultConfigFilePath

Reply:
go coding convention in [effective go](https://golang.org/doc/effective_go.html#Getters) doesn't recommend to use `Get` prefix.
Please see https://golang.org/doc/effective_go.html#Getters

I would recommend to use `DefaultConfigFilePath` without `Get` prefix.